### ISSUE TITLE
Viewer and photo overview enhancements

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/photos#27]
We should have the at least some of these functions 
1. in the picture view (accessable via sidebar?) and
2. in the overview with multiselect 

- [x] share
- [x] delete
- [x] download
- [ ] see (and change) info (dates, location)
- [x] favourite
- [ ] add to album

**Comment by skjnldsv** [comment]
I think this would be for server, add the full action menu in the sidebar ?
cc @jancborchardt

**Comment by jakobroehrl** [comment]
@skjnldsv Sidebar doen't show up at all in the viewer, see: https://github.com/nextcloud/photos/issues/22
Is this working for you?

**Comment by skjnldsv** [comment]
Nope, but the bug is listed here https://github.com/nextcloud/photos/issues/22, let's not mix up issues ;)

**Comment by jancborchardt** [comment]
Regarding

> in the picture view

see **Implement file actions menu https://github.com/nextcloud/viewer/pull/15** in the Viewer which has been open for a while. :)

So in this issue we should focus on how to do it in the view. But the priority is for sure to be able to do it in the viewer.

**Comment by skjnldsv** [comment]
> > in the picture view

Ah right, picture view = viewer.
I need vacations

**Comment by skjnldsv** [comment]
> So in this issue we should focus on how to do it in the view. But the priority is for sure to be able to do it in the viewer.

This requires a drastic change on server.
Not properly feasible in the current state, so for 19 maybe? :)

**Comment by jancborchardt** [comment]
If Gallery didn’t have it either (don’t think it did) then it’s fine to postpone to a later version. :)

**Comment by skjnldsv** [comment]
yes, this is something that should come with the files to vue :)
Having a proper API to get the list of available actions for files (without the files opened, meaning: in another app)

**Comment by codermehraj** [comment]
Great work

**Comment by skjnldsv** [comment]
@codermehraj thanks a lot ! :)
